src/project.cc

```diff
@@ -573,7 +580,7 @@ std::vector<Project::Entry> LoadCompilationEntriesFromDirectory(
 #ifdef _WIN32
     // TODO
 #else
-    unlink((comp_db_dir + "/compile_commands.json").c_str());
+    unlink((comp_db_dir + "compile_commands.json").c_str());
```

Comment:
Just curious, do you use this feature? I'm wondering if there are any people who depend on it.

Reply:
The symlink doesn't work on WIndows so I have to copy over the compile_commands.json file all the time from the CMake build directory to the project root (which is incredibly tedious). 

I was actually going to request removing the symlink from the cquery repository and have everyone use this method (since this should work everywhere).

Reply:
Sure, that's reasonable. Maybe we should have cquery look for a compile_commands.json in a build/* subfolder?

Reply:
That seems like a good idea, We should have a lot of cases covered by checking project root + build directory for compile_commands.json. Other directories are covered by CompilationDatabaseDirectory option

Reply:
Filed #586